docs/intro.txt

```diff
@@ -50,45 +50,53 @@ The :py:class:`AerBackend` also supports GPU simulation which can be configured
+
 ::
 
     from pytket.extensions.qiskit import set_ibmq_config
     
     set_ibmq_config(ibmq_api_token=ibm_token)
 
-This will save your IBMQ credentials locally. After saving your credentials you can access ``pytket-qiskit`` backend repeatedly without having to re-initialise your credentials.
+After saving your credentials you can access ``pytket-qiskit`` backend repeatedly without having to re-initialise your credentials.
 
 If you are a member of an IBM hub then you can add this information to ``set_ibmq_config`` as well.
 
 ::
 
     from pytket.extensions.qiskit import set_ibmq_config
 
-    set_ibmq_config(ibmq_api_token=ibm_token, hub='your hub', group='your group', project='your project')
+    set_ibmq_config(ibmq_api_token=ibm_token, instance=f"{hub}/{group}/{project}")
 
 Alternatively you can use the following qiskit commands to save your credentials
 locally without saving the token in pytket config:
 
+.. note:: If using pytket-qiskit 0.39.0 or older you will have to use the deprecated :py:meth:`IBMQ.save_account` instead of :py:meth:`IBMProvider.save_account` in the code below.
+
 ::
 
-    from qiskit import IBMQ
+    from qiskit_ibm_provider import IBMProvider
     from qiskit_ibm_runtime import QiskitRuntimeService
 
-    IBMQ.save_account(token=ibm_token)
+    IBMProvider.save_account(token=ibm_token)
     QiskitRuntimeService.save_account(channel="ibm_quantum", token=ibm_token)
 
-To see which devices you can access you can use the ``available_devices`` method on the :py:class:`IBMQBackend` or :py:class:`IBMQEmulatorBackend`. Note that it is possible to pass ``hub``, ``group`` and ``project`` parameters to this method. This allows you to see which devices are accessible through your IBM hub.
+To see which devices you can access you can use the ``available_devices`` method on the :py:class:`IBMQBackend` or :py:class:`IBMQEmulatorBackend`. Note that it is possible to pass optional ``instance`` and ``provider`` arguments to this method. This allows you to see which devices are accessible through your IBM hub.
 
 ::
 
     from pytket.extensions.qiskit import IBMQBackend
+    from qiskit_ibm_provider import IBMProvider
 
-    backend = IBMQBackend # Initialise backend for an IBM device
-    backend.available_devices(hub='your hub', group='your group', project='your project') 
+    my_instance=f"{hub}/{group}/{project}"
+    ibm_provider = IBMProvider(instance=my_instance)
+    backend = IBMQBackend("ibmq_belem") # Initialise backend for an IBM device
+    backendinfo_list = backend.available_devices(instance=my_instance, provider=ibm_provider) 
+    print([backend.device_name for backend in backendinfo_list])
```

Comment:
Do you both agree that the documentation here sufficently covers the changes the user will need to make when using 0.40.0?